main.py

```diff
@@ -5,322 +5,67 @@
-                        action = Action13.from_of_action(of_action)
-                        flow.actions.append(action)
-        return flow
-
-    @classmethod
-    def from_replies_flows(cls, flow04):
-        """Create a flow from a flow passed on
-        replies_flows in event kytos/of_core.flow_stats.received."""
-
-        flow = GenericFlow(version='0x04')
-        flow.idle_timeout = flow04.idle_timeout
-        flow.hard_timeout = flow04.hard_timeout
-        flow.priority = flow04.priority
-        flow.table_id = flow04.table_id
-        flow.cookie = flow04.cookie
-        flow.duration_sec = flow04.stats.duration_sec
-        flow.packet_count = flow04.stats.packet_count
-        flow.byte_count = flow04.stats.byte_count
-
-        as_of_match = flow04.match.as_of_match()
-        for match in as_of_match.oxm_match_fields:
-            match_field = MatchFieldFactory.from_of_tlv(match)
-            field_name = match_field.name
-            if field_name == 'dl_vlan':
-                field_name = 'vlan_vid'
-            flow.match[field_name] = match_field
-        flow.actions = []
-        for instruction in flow04.instructions:
-            if instruction.instruction_type == 'apply_actions':
-                for of_action in instruction.actions:
-                    flow.actions.append(of_action)
-        return flow
-
-    def do_match(self, args):
-        """Match a packet against this flow."""
-        if self.version == '0x04':
-            return self.match13(args)
-        return None
-
-    def match13(self, args):
-        """Match a packet against this flow (OF1.3)."""
-        # pylint: disable=consider-using-dict-items
-        for name in self.match.copy():
-            if name not in args:
-                return False
-            if name == 'vlan_vid':
-                field = args[name][-1]
-            else:
-                field = args[name]
-            if name not in ('ipv4_src', 'ipv4_dst', 'ipv6_src', 'ipv6_dst'):
-                if self.match[name].value != field:
-                    return False
-            else:
-                packet_ip = int(ipaddress.ip_address(field))
-                ip_addr = self.match[name].value
-                if packet_ip & ip_addr.netmask != ip_addr.address:
-                    return False
-        return self
 
 
 # pylint: disable=too-many-public-methods
 class Main(KytosNApp):
     """Main class of amlight/flow_stats NApp.
-
     This class is the entry point for this napp.
     """
 
     def setup(self):
         """Replace the '__init__' method for the KytosNApp subclass.
-
         The setup method is automatically called by the controller when your
         application is loaded.
-
         So, if you have any setup routine, insert it here.
         """
         log.info('Starting Kytos/Amlight flow manager')
-        for switch in self.controller.switches.copy().values():
-            switch.generic_flows = []
```

Comment:
@gretelliz you'll need to coordinate strategically when this PR will land, otherwise it'll break [`sdntrace_cp` ](https://github.com/kytos-ng/sdntrace_cp/blob/master/automate.py#L38), `sdntrace_cp` `Automate`'s related methods should also get the flows from `/flow_manager/v2/stored_flows`, so it'd be great to have a PR there to land these two shortly after both are approved.  

Also, `sdntrace_cp` used to store them in [`self._circuits`](https://github.com/kytos-ng/sdntrace_cp/blob/master/automate.py#L78), and it used to be updated on every event `amlight/flow_stats.flows_updated` that's being removed, so that can also be improved (removing that shared memory structure),  no need keep to subscribe to this event anymore, and then whenever it needs the flows it fetches from flow_manager as you've done and compute on-the-fly the transformations, since the scheduled methods won't run that frequent we shouldn't expect much surprises in terms of making too much requests.  Automated checks from sdntrace_cp are disabled by default, but we need to make sure it still works if in prod it's enabled.

Reply:
@viniarck, Good point, I hadn't realised. I have launched a [PR](https://github.com/kytos-ng/sdntrace_cp/pull/49)